src/ampform/dynamics/__init__.py

```diff
@@ -54,7 +54,7 @@ class EnergyDependentWidth(sp.Expr):
     m_b: Any
     angular_momentum: Any
     meson_radius: Any
-    phsp_factor: PhaseSpaceFactorProtocol = argument(
+    phsp_factor: PhaseSpaceFactorProtocol = argument(  # type:ignore[assignment]
```

Comment:
This is actually quite bad for downstream packages that lint with `mypy`. Since SymPy v1.13 (untyped), it `PhaseSpaceFactorProtocol` does not match the signature of any `sympy.Expr` classes.